spec/models/fields_of_operation_spec.rb

```diff
@@ -0,0 +1,14 @@
+RSpec.describe FieldsOfOperation do
+  describe "#links" do
```

Comment:
I think this should be `#fields_of_operation` as this is the method name

Reply:
Thanks, done 👍